README.md

```diff
@@ -1,2 +1,38 @@
 # hw01
 Repository for homework 1
+------------------------------
+# A bio for David Whyman!
+ 
+*David Whyman was born in Washington, DC on June 5, 1998.*
+
+### David's Education
+David graduated from Walt Whitman High School in Bethesda Maryland. Now, David is a public policy major at the University of Chicago.
+
+### David's Hobbies!
+* Travel
+* Literature
+  * Magical Realism
+* Art
+   * Figure Drawing
+   * Landscape Painting
+* Horseback Riding 
+
+### About David
+I look at the world in an analytical way. Deconstructing complex issues and seeing how all the little parts interact is really cool. That's why I'm curious about social science research. Computer science and statistics provide useful tools to analyze these issues. Oh and by the way I love dogs. Especially **golden retrievers**.
+
+### David's Activities
+At the University of Chicago, I am involved in the Paul Douglas Institute, *The Citizen Bulletin*, and *The Maroon*. Check out [this article](https://www.chicagomaroon.com/article/2018/3/30/preckwinkle-talks-soda-tax-criminal-justice/) I wrote for *The Maroon*.
+
+
+
+<img style="float:left;transform: rotate(0deg); width:300px" src="https://github.com/whymand/myrepo/blob/master/Biopic.jpg" />
```

Comment:
Not sure why you added the image here at the end. Could have added the image at the top of the page by using the following Markdown syntax with the image saved to your computer: ![](image link)